src/__tests__/suggestions.ts

```diff
@@ -400,11 +414,11 @@ test('getSuggestedQuery returns rich data for tooling', () => {
     variant: 'get',
   })
 
-  expect(getSuggestedQuery(button).toString()).toEqual(
+  expect((getSuggestedQuery(button) as Suggestion).toString()).toEqual(
```

Comment:
question: Is a non-null assertion sufficient?
```suggestion
  expect(getSuggestedQuery(button)!.toString()).toEqual(
```
To make sure we're not casting to something completely different.

Reply:
Yes it'll be ok :)